hooks/event-conversion-utils.ts

```diff
@@ -67,13 +68,31 @@ function getType (record: any): CommunityEventType {
   return 'Conference'
 }
 
-function getImage (_record: any): string {
-  const options = [
-    '/images/events/promo-finland-unconference.jpg',
-    '/images/events/promo-vermont.jpg',
-    '/images/events/promo-asia.jpg'
-  ]
-  return options[Math.floor(Math.random() * options.length)]
+function getImage (record: any): string {
+  const fallbackImage = '/images/events/no-picture.jpg'
+  const attachments = record.get(RECORD_FIELDS.image)
+  const imageAttachment = attachments && getImageAttachment(attachments)
+  const imageUrl = imageAttachment && getBestSizeUrl(imageAttachment)
+  return imageUrl || fallbackImage
+}
+
+function getImageAttachment (attachments: any[]): any|null {
+  for (const oneAttachment of attachments) {
+    if (oneAttachment.type.startsWith('image')) {
+      return oneAttachment
+    }
+  }
+  return null
+}
+
+function getBestSizeUrl (imageAttachment: any): string {
```

Comment:
Suggestion to improve legibility:
```
function getImageUrl (imageAttachment: any): string {
    const { thumbnails } = imageAttachment
    const { large: largeThumbnail } = thumbnails
    const noThumbnailsForImage = !thumbnails
    const noLargeThumbnailsForImage = !largeThumbnail

    if (noThumbnailsForImage || noLargeThumbnailsForImage) {
        return imageAttachment.url
    }

    return largeThumbnail.url
}
```

Reply:
I cannot destructure `undefined` in case it does not exist, but I get the idea.